### PR TITLE
Update Decision Review Created Message Generator

### PR DESCRIPTION
**Pull request by chrislazo22**
Resolves [Update Decision Review Created Message Generator to Utilize Different Veterans for Different Scenarios](https://jira.devops.va.gov/browse/APPEALS-50300)

# Description
Update generator to create a variety of veterans for different use case scenarios so that testing covers sufficient scenarios. Updated Generator to start with prefix and to increment 1 each time. Updated Randomize_value to new method update_value. Updated Rspec test to reflect changes all pass

## Acceptance Criteria
- [x] Generator utilizes unique veterans for each test case

## Testing Plan

1. Go to [Test Update Decision Review Created Message Generator](https://jira.devops.va.gov/browse/APPEALS-51194)
2. [Test Exec X-RAY](https://jira.devops.va.gov/secure/XrayExecuteTest!default.jspa?testExecIssueKey=APPEALS-51554&testIssueKey=APPEALS-51194)
3. [Test Exec](https://jira.devops.va.gov/browse/APPEALS-51554)

### Test Coverage
Did you include any test coverage for your code? Check below:
- [x] RSpec

### Code Climate
Your code does not add any new code climate offenses? If so why?
- [x] No new code climate issues added

### CODE REVIEW

File: lib/kafka_message_generators/decision_review_created_events.rb [hunk 1]
```diff
@@ -471,6 +471,7 @@ def create_drc_message_without_bis_person(issue_trait, code)
     # represents a message post-consumption and post-deserialization
     # some fields must be changed to accurately reflect a message prior to consumption
     def create_drc_message(trait, ep_code)
+      # something needs to change here
```
Comment (Aaron-Willis) [inline lib/kafka_message_generators/decision_review_created_events.rb:474]: Is this code comment still valid?

File: lib/kafka_message_generators/decision_review_created_events.rb [hunk 2]
```diff
@@ -472,7 +462,7 @@ def create_drc_message_without_bis_person(issue_trait, code)
     # some fields must be changed to accurately reflect a message prior to consumption
     def create_drc_message(trait, ep_code)
       drc = FactoryBot.build(:decision_review_created, trait.to_sym, ep_code: ep_code)
-      randomize_claim_id(drc)
+      update_claim_id(drc, trait)
```
Comment (chrislazo22) [inline lib/kafka_message_generators/decision_review_created_events.rb:465]: Trait does not need to be passed anymore

Comment (Jruuuu) [reply lib/kafka_message_generators/decision_review_created_events.rb]: removed

**Review by Jruuuu — COMMENTED** [review]
LGTM

**Review by Aaron-Willis — APPROVED** [review]

**Pull request merged**